multichain-testing/README.md

```diff
@@ -0,0 +1,63 @@
+# Starship 
+
+End-to-end testing environment for fully simulated chains, powered by [Starship](https://docs.cosmology.zone/starship).
+
+
+## Configuration
+
+The current commands will read from [`config.yaml`](./config.yaml) to build a multi-chain teting environment. Currently, the image includes `agoric`, `osmosis`, and `cosmos-hub` chains and a hermes relayer between each.
+
+A less-resource intensive version of the configuration is available at [`config.ci.yaml`](./config.ci.yaml) and is currently used for CI tests.
```

Comment:
_question_

Do we want to maintain a `config.ci.yaml` file or do we just use the same `config.yaml` for both dev laptop and github action and bump up github action runner size to accommodate?

Reply:
Would love to give the latter a go. Will push a commit that does that (and disables the ping.pub explorer for now). Are you able to adjust the GH runner size for this job?

Reply:
This helped big time - thank you! As of [7490f0f](https://github.com/Agoric/agoric-sdk/pull/9462/commits/7490f0fc33e71ef9abd65e896e0b278a201da8b1) and [job 26](https://github.com/Agoric/agoric-sdk/actions/runs/9422008490/job/25957334476#step:9:22) currently trying to figure out why the chain registries aren't available

Reply:
Solved by updating to starship-action@0.3.x!